lib/nylas/model.rb

```diff
@@ -99,6 +109,10 @@ def save_call
       )
     end
 
+    def query_params
+      {}
```

Comment:
If you're initializing `query_params` as `{}` by default in line 42, do you still need this method?

Reply:
Yes we need to as this `Model` class is used in other places like events and message where we can override this.

Also `query_params` is used https://github.com/nylas/nylas-ruby/blob/support-notify-participants-on-event/lib/nylas/model.rb#L53